src/pages/About.tsx

```diff
@@ -61,7 +61,7 @@ const About = withStyles(styles)(({ classes }: AboutProps) => (
       <Typography variant="h5">
         Förbundsstyrelsen <Emoji emoji="😍" />
       </Typography>
-      <Grid container justify="space-around" spacing={2}>
```

Comment:
I think this unfortunately makes the space have extra spacing, causing a horizontal scrollbar to appear.
<img width="840" alt="image" src="https://github.com/digitalungdom-se/digitalungdom.se/assets/28629647/478e81f7-da69-4d96-b198-e21b04f56cbc">

Reply:
I have reverted this change, you can fix it later on if you want to @sakerhetspolisen.